README.md

```diff
@@ -59,58 +74,54 @@ What | Type | Description
+Configuration can be applied by passing a `Configuration` object to the `BarcodeScannerRenderer.Init()` method. As the available options are platform specific, the configuration has to be done in the according platform solution. The corresponding [Android](Rb.Forms.Barcode.Droid/Configuration.cs) class documentation should give you a solid understanding of the available options. 
 
-By default most of the options are disabled to ensure the highest device compatibility. 
+By default the compatibility mode is enabled to ensure the highest device compatibility. 
 
 Simple example:
 
-    var rbConfig = new Rb.Forms.Barcode.Droid.RbConfig {
-        TryHarder = true
+    var config = new Configuration {
+        // Some devices, mostly samsung, stop auto focusing as soon as one of the advanced features is enabled.
+        CompatibilityMode = Build.Manufacturer.Contains("samsung")
     };
-    BarcodeScannerRenderer.Init(rbConfig);
+
+    BarcodeScannerRenderer.Init(config);
 
 ### Debugging
 
-Rb.Forms.Barcode provides you with a tremendous amount of debug information, so do not forget to check your application log:
+Rb.Forms.Barcode provides you with a tremendous amount of debug information, so check your application log if anything goes wrong:
 
 ```
 [Rb.Forms.Barcode] [BarcodeScannerRenderer] OnElementChanged
 [Rb.Forms.Barcode] [BarcodeScannerRenderer] OnElementPropertyChanged
 [Rb.Forms.Barcode] [BarcodeScannerRenderer] SurfaceCreated
-[Rb.Forms.Barcode] [CameraControl] OpenCamera
-[Rb.Forms.Barcode] [CameraControl] AssignPreview
 [Rb.Forms.Barcode] [BarcodeScannerRenderer] SurfaceChanged
-[Rb.Forms.Barcode] [CameraControl] StartPreview
 [Rb.Forms.Barcode] [CameraConfigurator] Focus Mode [continuous-picture]
 [Rb.Forms.Barcode] [CameraConfigurator] Scene Mode [auto]
 [Rb.Forms.Barcode] [CameraConfigurator] Metering area [True]
-[Rb.Forms.Barcode] [CameraConfigurator] Video stabilization [False]
+[Rb.Forms.Barcode] [CameraConfigurator] Focusing area [True]
+[Rb.Forms.Barcode] [CameraConfigurator] Video stabilization [True]
 [Rb.Forms.Barcode] [CameraConfigurator] White balance [auto]
-[Rb.Forms.Barcode] [BarcodeScannerRenderer] OnVisibilityChanged [Gone]
-[Rb.Forms.Barcode] [CameraControl] HaltPreview
+[Rb.Forms.Barcode] [BarcodeScannerRenderer] OnElementPropertyChanged
+[ScannerView] OnBarcodeChanged [886970911399 - UpcA]
+[ScannerView] OnBarcodeDecoded [886970911399 - UpcA]
+Decoded barcode [886970911399 - UpcA]
 [Rb.Forms.Barcode] [BarcodeScannerRenderer] SurfaceDestroyed
-[Rb.Forms.Barcode] [CameraControl] ReleaseCamera
+[Rb.Forms.Barcode] [BarcodeScannerRenderer] OnElementPropertyChanged
+[Rb.Forms.Barcode] [BarcodeScannerRenderer] Enabled [False]
+[Rb.Forms.Barcode] [BarcodeScannerRenderer] OnElementPropertyChanged
 [Rb.Forms.Barcode] [BarcodeScannerRenderer] Disposing
 ```
 
```

Comment:
Nice.  In the future it would make sense to have the log-level also be configurable in `Configuration.cs` because it's not uncommon for people to want to turn off logs in production apps.  See last line of intro here:

https://github.com/JakeWharton/timber